ast/src/main/java/com/graphicsfuzz/common/glslversion/ShadingLanguageVersion.java

```diff
@@ -177,6 +177,8 @@ static ShadingLanguageVersion webGlFromVersionString(String versionString) {
 
   boolean supportedIntBitsToFloat();
 
```

Comment:
Can you add a JavaDoc comment for this new method, explaining what 'integer functions' means with regards to the spec?